package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "font.js",
+  "version": "1.0.0",
+  "description": "What if you could actually inspect your web fonts? In the same context that you actually use those web fonts? That's what this is for.",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Pomax/Font.js.git"
+  },
+  "keywords": [
+    "font",
+    "ttf",
+    "woff",
+    "woff2",
+    "otf",
+    "fontkit",
+    "opentype"
+  ],
+  "author": "Pomax",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Pomax/Font.js/issues"
+  },
+  "homepage": "https://github.com/Pomax/Font.js#readme",
+  "type": "module",
+  "dependencies": {
+    "fs": "0.0.1-security"
+  }
```

Comment:
```suggestion
  "homepage": "https://github.com/Pomax/Font.js#readme"
```

Reply:
In an older version of FontNode, the system was complaining about a missing fs module. Now it is working also without.